EFCore.VisualBasic/Design/Internal/VisualBasicHelper.vb

```diff
@@ -803,7 +796,21 @@ Namespace Design.Internal
         End Function
 
         Private Function Fragment(frag As NestedClosureCodeFragment) As String
-            Return $"Function({frag.Parameter}) {frag.Parameter & Fragment(frag.MethodCall)}"
+            If frag.MethodCalls.Count = 1 Then
+                Return $"Function({frag.Parameter}) {frag.Parameter}{Fragment(frag.MethodCalls(0))}"
+            End If
+
+            Dim builder = New IndentedStringBuilder()
+            builder.AppendLine($"Function({frag.Parameter})")
+            Using builder.Indent()
+                For Each methodCall In frag.MethodCalls
+                    builder.AppendLine(frag.Parameter & Fragment(methodCall))
+                Next
+            End Using
+
+            builder.AppendLine("End Function")
+
+            Return builder.ToString()
         End Function
```

Comment:
Should be sufficient for support for adding / removing / modifying entities mapped to a temporal table
- #37